addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-markdown.html

```diff
@@ -7,7 +7,12 @@
 	<H1>Traditional Markdown Report</H1>
 
 	<H3>Sample</H3>
-
+	<H4>About riskdesc</H4>
+	riskdesc - Is a combination identifier, showing Risk followed by
+	Confidence (in brackets). For example
+	<code>High (Medium)</code>
+	as shown in example below, would indicate a High risk issue identified
```

Comment:
Why is this one different in all cases there's an example?

Reply:
> Why is this one different in all cases there's an example?

I did that because the Riscdesc in that particular template is very far down the file and it seemed weird to be at the top without more information. I can change back to the others though.

Reply:
Agreed this one needs to be explained differently than the others (since it's not a named field), how about:
```html
<H4>Header <code>Risk (Confidence)</code></H4>
This header is a combination identifier, showing Risk followed by Confidence. For example <code>High (Medium)</code>, would indicate a High risk issue identified with Medium confidence.
```

Reply:
Yeah, I can do that.

Reply:
Thank you both.

Reply:
I appreciate both your patience. I am glad I started with something simple, the version control of this is a big learning curve.

Reply:
So if I have altered 2 files since the last push (Changelog and the template file to accomodate thc202 wording), is it the same commit process?

git rebase
./gradlew :aO:reports:spotlessApply
git commit -a --amend --no-edit --signoff 
git push --force`

Reply:
Shouldn’t need the rebase, bottom 3 steps are good.

Reply:
Thanks. I am getting the following error: 

`$ git commit -a --amend --no-edit --signoff 
fatal: this operation must be run in a work tree`

Shoulld I exlude the --amend?

Reply:
Could you provide the output from `git status`? I don't understand why it thinks you're mid-merge.

Reply:
sure...

`On branch doc-changes
Your branch is up to date with 'origin/doc-changes'.

All conflicts fixed but you are still merging.
  (use "git commit" to conclude merge)

Changes to be committed:
        modified:   addOns/reports/CHANGELOG.md
        modified:   addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-markdown.html
        modified:   addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-xml-plus.html

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-markdown.html
        modified:   addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-xml-plus.html

`

Reply:
Did you actually start with a rebase?

Reply:
That'd explain why it thinks you're merging.

Reply:
I did a git pull but not a rebase

Reply:
Maybe my thoughts on how this works is not correct but I assume I need to grab the current version of the repo before I do anything?

Reply:
Okay pull is the same as fetch and merge. That's why.

You're better off doing a fetch and rebase, that way you don't end up with unrelated commits in your PR.

For the time being I'd suggest `git merge --abort` and go ahead with what you're trying to do.